library.properties

```diff
@@ -1,5 +1,5 @@
 name=Arduino_PF1550
-version=0.2.0
+version=0.3.0
```

Comment:
Incrementing a libraries version number is usually a separate step done outside of a pull-request via direct commit.